### DISCUSSION TITLE
Multiple filters

### DISCUSSION THREAD

**Discussion by gregdingle** [AntlerVC/firetable#1078] [Idea / Feature Request]
It's useful to filter on multiple columns. I realize this sometimes requires an index. The firestore console allows it. 

![CleanShot 2023-01-12 at 18 02 12](https://user-images.githubusercontent.com/28797/212220072-4bf9fa33-9627-4af9-8c63-958271420626.png)

**Comment by BeeBombshell** [comment]
Hey @gregdingle, I think having multiple filter options would be pretty interesting. We'd love for you to discuss your ideas with our team and community during our Bi-weekly Office Hours. Here's a link to register: https://lu.ma/rowy-demoday-01

**Reply by gregdingle** [reply]
@BeeBombshell I'd like to give some feedback about that and a bunch more stuff but the event has already passed and it was at 3:30AM my time anyway. 

Maybe we could schedule a session directly with your team? I promise it will be worth your time.

**Reply by harinij** [reply]
Hi @gregdingle, I would love to chat. Can you book a time that works for you [here](https://rowy.io/officehour).

**Comment by bofeiw** [comment]
Multiple filters is supported in the [Rowy 3.0 release](https://github.com/rowyio/rowy/releases/tag/v3.0.0).

![image](https://github.com/rowyio/rowy/assets/34177142/594adf17-4a81-4161-96ab-a7ccf458c2c4)
